Console/Command/NostoAccountRemoveCommand.php

```diff
@@ -82,18 +82,18 @@ class NostoAccountRemoveCommand extends Command
      * @param NostoAccountHelper $nostoAccountHelper
      * @param NostoScopeHelper $nostoScopeHelper
      * @param WriterInterface $appConfig
-     * @param NostoHelperCache $nostoHelperCache
+     * @param NostoHelperCache $nostoCacheHelper
```

Comment:
I think this makes the readability a little bit confusing. Maybe change back or change both vars?